packages/reporter/interfaces.d.ts

```diff
@@ -11,6 +11,7 @@ interface MatchCommandResult {
 export type TestContextResult = Omit<ReportData, 'matchers'>;
 
 export interface ReportData {
+  suiteName?: string;
```

Comment:
I guess we should have same structure for all test frameworks. How I can see there is two ways to solve it:
1. Implement `suiteName` for jasmine and mocha too in this PR
2. Remove `suiteName` and implement this one for all test frameworks in new PR

What do you think about this?

Reply:
I thought you might comment on that :) 

I did a quick research. At least with jasmine we only get the following:

```javascript
{
  id: 'spec0',
  description: 'Should check element',
  fullName: 'Visual regression Should check element',
  failedExpectations: [],
  passedExpectations: [
    {
      matcherName: 'toEqual',
      message: 'Passed.',
      stack: '',
      passed: true
    }
  ],
  deprecationWarnings: [],
  pendingReason: '',
  duration: 219,
  start: 1584043081114,
  title: 'Should check element',
  fullTitle: 'Visual regression Should check element',
  error: undefined,
  passed: true
}
```

As there is no structural info for the suite I implemented a quick option to extract it from the fullTitle. It works for jasmine, I guess it does as well for mocha.

Reply:
There may be case when describe will be nested, for example.
```ts
describe('Level 1', () => {
  describe('Level 2', () => {
    it('Some test case here', () => {})
  })
})
```
So, `fullTitle` will be `Level 1 Level 2 Some test case here` and it's not expected behaviour. I guess we should create new PR and discuss the possible solutions.

Reply:
Done.